### PR TITLE
Update kubelet version to fix issue of Resource Handling Configuration

### PR DESCRIPTION
**Pull request by ydye**
#915

Update:
After investigating, I found the build version 1.9.4 of kubelet docker image has some issue which may lead to image-gc-threshold failure.

### CODE REVIEW

File: pai-management/k8sPaiLibrary/template/kubelet.sh.template [hunk 1]
```diff
@@ -44,7 +44,6 @@ docker run \
   --pod-manifest-path=/etc/kubernetes/manifests \
   --allow-privileged=true \
   --logtostderr=true \
-  --image-gc-high-threshold=98 \
-  --image-gc-low-threshold=95 \
   --pod-infra-container-image {{ clusterconfig['dockerregistry'] }}/pause-amd64:3.0 \
+  --eviction-hard=memory.available<5%,nodefs.available<5%,imagefs.available<5%,nodefs.inodesFree<5%,imagefs.inodesFree<5% \
```
Comment (fanyangCS) [inline pai-management/k8sPaiLibrary/template/kubelet.sh.template]: do you know why the previous config doesn't work? can we add a comment to state which version of kubelet support the flag?

Comment (ydye) [reply pai-management/k8sPaiLibrary/template/kubelet.sh.template]: Flag --image-gc-high-threshold has been deprecated, This parameter should be set via the config file Flag --image-gc-low-threshold has been deprecated, This parameter should be set via the config file

Comment (ydye) [reply pai-management/k8sPaiLibrary/template/kubelet.sh.template]: Some kubelet Garbage Collection features in this doc will be replaced by kubelet eviction in the future.

REF: https://kubernetes.io/docs/concepts/cluster-administration/kubelet-garbage-collection/

**Comment by YitongFeng** [comment]
May I ask what's the issue? @ydye

**Comment by ydye** [comment]
@YitongFeng  Didn't find it on the changelog of 1.9

**Pull request merged**